frontend/src/pages/landing/landing.tsx

```diff
@@ -0,0 +1,39 @@
+import { type FC } from 'react';
+
+import { Link } from '~/libs/components/components.js';
+import { AppRoute } from '~/libs/enums/app-route.enum';
```

Comment:
```suggestion
import { AppRoute } from '~/libs/enums/enums.js';
```

Reply:
[e786fcb](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/26/commits/e786fcbde5dad171ab47b209f69f0186f52d4f9b)

I would like webstorm to choose the right paths on its own. But I have not yet found how to configure it correctly, especially when there are mono repo + aliases. Something goes wrong at once, it can't decide, or it substitutes from the primary source.